troubleshooting.md

```diff
@@ -0,0 +1,32 @@
+# Trouble shooting
+
+## Possible FAQs
+1.  How to make sure my web page with this extension use redirection when connect to Azure MySQL?  
```

Comment:
How do I ensure my application uses redirection with this extension to connect to Azure Database for MySQL?